sap-lama-apps/azuredeploy.json

```diff
@@ -0,0 +1,622 @@
+            },
+            "defaultValue": ""
+        }
+    },
+    "variables": {
+        "images": {
+            "Windows Server 2012 Datacenter": {
+                "sku": "2012-Datacenter",
+                "offer": "WindowsServer",
+                "publisher": "MicrosoftWindowsServer",
+                "OSType": "Windows",
+                "osDiskSize": 128
+            },
+            "Windows Server 2012 R2 Datacenter": {
+                "sku": "2012-R2-Datacenter",
+                "offer": "WindowsServer",
+                "publisher": "MicrosoftWindowsServer",
+                "OSType": "Windows",
+                "osDiskSize": 128
+            },
+            "Windows Server 2016 Datacenter": {
+                "sku": "2016-Datacenter",
+                "offer": "WindowsServer",
+                "publisher": "MicrosoftWindowsServer",
+                "OSType": "Windows",
+                "osDiskSize": 128
+            },
+            "SLES 12": {
+                "sku": "12-SP3",
+                "offer": "SLES-SAP",
+                "publisher": "SUSE",
+                "OSType": "Linux",
+                "osDiskSize": 128
+            },
+            "RHEL 7": {
+                "sku": "7.3",
+                "offer": "RHEL",
+                "publisher": "RedHat",
+                "OSType": "Linux",
+                "osDiskSize": 128
+            },
+            "Oracle Linux 7": {
+                "sku": "7.3",
+                "offer": "Oracle-Linux",
+                "publisher": "Oracle",
+                "OSType": "Linux",
+                "osDiskSize": 128
+            }
+        },
+        "internalOSType": "[variables('images')[parameters('osType')].OSType]",
+        "osConfigs": {
+            "Windows": {
+                "Publisher": "Microsoft.Compute",
+                "Name": "CustomScriptExtension",
+                "Version": "1.7",
+                "script": "[concat(parameters('_artifactsLocation'), '/scripts/diskConfig.ps1', parameters('_artifactsLocationSasToken'))]",
+                "scriptCall": "powershell.exe -ExecutionPolicy bypass -File diskConfig.ps1",
+                "disks": [
+                    {
+                        "lun": 0,
+                        "name": "[concat(parameters('computerName'), '-', variables('sidlower'), '-apps')]",
+                        "caching": "ReadOnly",
+                        "createOption": "Empty",
+                        "diskSizeGB": 128
+                    }
+                ],
+                "scriptArguments": "[concat('-luns \"0\" -names \"sap\" -paths \"C:\\usr\\sap', parameters('sapSystemId'), '\"  -sizes \"100\"')]"
+            },
+            "Linux": {
+                "Publisher": "Microsoft.Azure.Extensions",
+                "Name": "CustomScript",
+                "Version": "2.0",
```

Comment:
set autoupgrademinorversion to true

Reply:
The VM extensions that use these variables have "autoUpgradeMinorVersion": true,